spec.md

```diff
@@ -148,30 +148,34 @@ that contains both context and data).
 
 ### source
 * Type: Object
-* Description: This describes the software instance that emits the event at
-  runtime (i.e. the producer). It contains sub-properties (listed below)
+* Description: This describes the software where an event occurred. It contains
+  sub-properties (listed below)
 * Constraints:
   * REQUIRED
   * MUST contain at least one non-empty sub-property.
 
 ### source-type
 * Type: String
-* Description: Type of the event source. Providers define list of event
-  sources.
+* Description: The kind of object on which an event occurred. Providers
+  define list of source-types.
 * Constraints:
   * REQUIRED
   * MUST be a non-empty string
 * Examples:
-  * s3
+  * aws:s3:object
```

Comment:
ALERT AWS USERS:
I previously avoided touching the AWS samples because I'm a Google employee and don't want to give the impression that I'm dictating what Amazon should do. @ultrasaurus pointed out that leaving the sample untouched made the examples incongruous so I made one up. I'd love an Amazon employee or pro to help me here. My thought process:

1. Amazon seems to be relatively inconsistent between using an "aws:" prefix (e.g. ARNs) or not (e.g. event types). I chose the former on a whim.
2. Amazon clearly prefers colon separators to period separators.
3. Since all S3 event types are on objects, the source-type here was an object, not an "s3" (bucket).
4. I couldn't find a policy "type" in Amazon for an S3 object; Amazon IAM uses <verb><noun> nomenclature like "s3:PutObject", so I made up a type string using the noun ("object") that their API uses.